packages/FPauth-strategies/FPauth-strategies.1.0.0/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+license: "MIT"
+synopsis: "Strategies to be used with FPauth-core authentication system"
+maintainer: "Mikhail Geine <mike.geine@gmail.com>"
+authors: [
+  "Mikhail Geine <mike.geine@gmail.com>"
+  "Pavel Argentov <argentoff@gmail.com>"
+]
+homepage: "https://github.com/mikeGEINE/FPauth"
+bug-reports: "https://github.com/mikeGEINE/FPauth/issues"
+depends: [
+  "dream" {>= "1.0.0~alpha3"}
+  "argon2"
+  "twostep"
+  "FPauth-core" {= version}
+  "ocaml" {>= "4.12.0"}
+  "dune" {>= "2.7"}
+  "base"
+  "alcotest" {with-test}
+  "bisect_ppx" {with-test & >= "2.5.0"}
+  "odoc" {with-doc}
+]
+depexts: [["libargon2-0"]|["libargon2-1"]|["argon2-libs"]]
```

Comment:
This is not the syntax for depexts. See https://github.com/ocaml/opam-repository/pull/18188 as an example

Reply:
It would also be nice if these were confined to a `conf-libargon2` package or something like that. This way the more frequent changes to depexts does not have to be repeated on all the releases

Reply:
Thank you for your advice ;)
I've added the package, but I'm still trying to deal with some distros, working on it...